doc/src/modules/functions/elementary.rst

```diff
@@ -1,78 +1,130 @@
 Elementary
 ==========
 
-This module implements elementary functions, as well as functions like ``Abs``,
-``Max``, etc.
+This module implements elementary functions such as trigonometric, hyperbolic
+as well as functions like ``Abs``, ``Max``, ``sqrt`` etc.
```

Comment:
> If that's the correct definition, then sqrt need not be explicitly mentioned because it's already elementary.

`sqrt` is mentioned just to give a brief idea of what it contains.